contents/docs/user-guides/support.md

```diff
@@ -0,0 +1,143 @@
+
+The objective is that you spend as little time as possible worrying about deploying PostHog.
+
+## Configuration assistance
+
+We help you define meaningful dashboards for your company and regularly check in that the configuration of PostHog is helping you improve your product.
+
+We ask to be a user in your PostHog team so we can suggest ways to get more value out of the product. Typically this means ensuring teams build out:
+
+* Dashboards that cover Acquisition, Activation, Retention, Referral and Revenue
+* Deep dive dashboards to help you understand fluctuations in the above
+* Give guidance on using session recording, heatmaps and paths appropriately (where relevant), specific to your product
+* Help you understand the impact on your core metrics of changes deployed behind feature flags
+
+## Standard Terms and Conditions
+
+PostHog's open source product is MIT-licensed.
+
+PostHog Cloud, PostHog Scale and PostHog Enterprise have [standard terms](/terms).
+
+If you need non-standard terms or Service Level Agreements (SLAs), for example, if you need to run a legal review of our terms and anticipate requesting changes, we can offer a Master Services Agreement for Enterprise customers.
+
+## Master Services Agreement (MSA)
+
+We can offer a Master Services Agreement to Enterprise customers to cover:
+
+* Bespoke pricing
+* Custom Service Level Agreements
+* Custom legal requirements (if your legal team wants to review)
+* Custom information security requirements
+
+We will _not_ automatically accept all changes, but we will be willing to review changes on our Enterprise product. For our other products, we will not do this since the contract sizes are generally much smaller.
+
+## Security assessment
+
+We have an extensive range of internal [security policies](/handbook/company/security) that apply to all products.
+
+For Enterprise customers, we are happy to complete security assessments. This can take the form of us enabling your team internally (who are responsible for info security since they're hosting it), or it can mean us just answering questions, or even adapating our policies if required and mutually agreed.
+
+## Bespoke pricing
+
+At this time we can offer bespoke pricing for Enteprise customers.
+
+For example, we have been asked if we can offer a set fee with a large usage cap rather than usage-based pricing, in order to make budget approval simpler.
+
+## Payment via invoicing
+
+We prefer to take payments via card (we use Stripe to process them). However, for companies with invoices over $2,000/month, we are able to support manual invoicing if needed. Just email billing@posthog.com and we'll set this up.
```

Comment:
Do we need to mention how we accept the payments (e.g. not cheques?)

Reply:
I think that's more a subset of invoicing than a "support doc" type level thing. We could clarify somewhere else in future but not a big deal